homeassistant/components/usb/__init__.py

```diff
@@ -31,7 +32,8 @@
 REQUEST_SCAN_COOLDOWN = 60  # 1 minute cooldown
 
 
-class UsbServiceInfo(TypedDict):
+@dataclasses.dataclass
+class UsbServiceInfo:
```

Comment:
We need to add inheritance when #60136 is merged
```suggestion
class UsbServiceInfo(BaseServiceInfo):
```

Reply:
Updated following rebase.